### PR TITLE
Small control inputs and input groups

### PR DESCRIPTION
**Pull request by jrubenoff**
Support small inputs for checkbox / radio buttons, and input groups.

See http://dvlcore.dev/forms for examples.

### CODE REVIEW

**Comment by jrubenoff** [comment]
Also, @ajb, if you're curious as to when small inputs should be used, I added a note to the style guide.

File: spec/dummy/app/views/home/forms.rb [hunk 1]
```diff
@@ -228,8 +229,13 @@ def main
                 as: :dropdown_select,
                 input_html: { class: 'small' }
 
+        label.control.control_small {
```
Comment (ajb) [inline spec/dummy/app/views/home/forms.rb]: Here you use `control_small`, but in the CSS you're just using `small`

File: spec/dummy/app/views/home/forms.rb [hunk 2]
```diff
@@ -2,7 +2,8 @@ class Views::Home::Forms < Views::Page
   def main
     h2 'Forms'
 
-    h3 'Form elements'
+    h3 'For
```
Comment (ajb) [inline spec/dummy/app/views/home/forms.rb]: Hm?

**Comment by ajb** [comment]
Cool, looks good. This still needs help though, even after I fixed the class names:

img

**Comment by jrubenoff** [comment]
I wanted to keep the checkbox the same size so that it's not more difficult to press.

File: spec/dummy/app/views/home/forms.rb (hunk 1 above)
Comment (jrubenoff) [inline spec/dummy/app/views/home/forms.rb]: My mistake. Fixed

**Comment by ajb** [comment]
OK, you might need to change the line height for proper vertical alignment?

**Comment by jrubenoff** [comment]
Ahhh, thanks. Done.

**Pull request merged**